frontend/src/app/pages/exercises/exercise/shared/trainer-map-editor/trainer-map-editor.component.html

```diff
@@ -18,6 +18,31 @@ <h5>Ansicht/Transferpunkt</h5>
                         class="overflow-auto w-100"
                         style="white-space: nowrap"
                     >
+                        <div
+                            class="d-inline-block card text-center"
+                            style="width: 10rem; text-overflow: ellipsis"
+                            (mousedown)="
+                                dragElementService.onMouseDown($event, {
+                                    type: 'simulatedRegion',
+                                    template: simulatedRegionTemplate
+                                })
+                            "
+                        >
+                            <div class="preview-image-container bg-secondary">
+                                <img
+                                    [src]="simulatedRegionTemplate.image.url"
+                                    class="card-img-top"
+                                />
+                            </div>
+                            <div class="card-body">
+                                <h5
+                                    class="card-title text-truncate"
+                                    [title]="'Simulierter Bereich'"
+                                >
+                                    Simulierter Bereich
+                                </h5>
+                            </div>
+                        </div>
```

Comment:
Not a big thing, but do we want to place the simulated regions in front of all existing features (viewport and transfer point)? Especially since the simulated regions are not ready to use yet.

Instead of

1. placing it behind the viewport and transfer point

we could also

2. introduce a new headline for this (would make sense especially if we'd like to have the different types (Patientenablage, B-Raum) as separate elements in the editor)
3. add tabs to the editor so that there is one "page" with all features needed for the existing concept and one "page" with all features needed for the new concept (there would be some overlap, for example transfer points and patients) but there are a lot of features which are not needed in one of the different modes.

If we choose the third option, I'd suggest to do this in another issue.

What do you think about this?

Reply:
I've implemented 2.